src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/Card.java

```diff
@@ -7,7 +7,7 @@
 import javax.persistence.*;
 
 @Entity
-public class Card implements Comparable<Card>, BelongsToGame {
+public class Card implements Comparable<Card>, BelongsToGame, IsOwnCard {
```

Comment:
```suggestion
public class Card implements Comparable<Card>, BelongsToGame {
```
The interface IsOwnCard is unused now. (And does not say anything and will not be used)